app/views/movies/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1> Movie Show Page </h1>
+
+<section id="movie_info-<%= @movie.id %>">
+  <h4><%= @movie.title %></h4>
+  <p>Year Made: <%= @movie.creation_year %></p>
+  <p>Genre: <%= @movie.genre %></p>
+</section>
+
+<section id="movie_actors-<%= @movie.id %>">
+
+<h4>Movie Actors<h4>
+<% @movie.orders_actors_by_age.each do |actor| %>
+<ul>
+
+<li>Name: <%= actor.name %></li>
+<li>Age: <%= actor.age %></li>
+</ul>
+<% end %>
+
+<p> Average Actor's Age: <%= @movie.average_age.to_i %> years. </p>
```

Comment:
Consider adding `to_i` to the end of your model method to encapsulate all of the average calculation in one place. While rounding it does have to do with "presentation," it's still somewhat connected to the average calculation which I'd say is business logic for the model to take care of (This is a very minor)